### DISCUSSION TITLE
User defined map area

### DISCUSSION THREAD

**Discussion by arrfonseca** [domlysz/BlenderGIS#516] [General]
Sorry if the question is already answered. I was wondering if it is possible to draw a square area inside the map screen to define the map sector you want to import. I understand that is only possible to import the whole map inside the zoom view, but I wanted a rectangular (taller than wide) section of that zoom that won't fit in another zoom level. Besides, at the zoom level I can actually see the entire city, for example, there is too much data for importing buildings and roads.

**Comment by domlysz** [comment]
Can't you cut the resulting mesh as you want afterward ?
